src/components/complete-profile-second/work-form/validation.ts

```diff
@@ -1,7 +1,7 @@
-import { ObjectSchema, boolean, date, object, string } from 'yup';
-import { useLocales } from 'src/locales';
-import { MAX_CHARACTERS_LENGTH } from 'src/constants';
 import { CompleteProfileSecondValues } from 'src/components/complete-profile-second/types';
+import { MAX_CHARACTERS_LENGTH } from 'src/constants';
+import { useLocales } from 'src/locales';
+import { ObjectSchema, boolean, date, object, string } from 'yup';
```

Comment:
1.1.14 import rules. imports should come in a specific order: node modules, absolute path, relative path

Reply:
fixed